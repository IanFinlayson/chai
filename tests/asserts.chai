def main():
    assert 2 * 3 > 5
    assert False or True
    assert 3 < 4
    assert "hey" in "what are they doing?"
    assert "i" in "Ian"
    print("not gonna get here...")
    
