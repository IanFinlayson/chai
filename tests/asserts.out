Asserton failed: "i" in "Ian"
